### PR TITLE
replacing manual command-line parsing with docopt

### PR DESCRIPTION
**Pull request by ShawnMilo**
Updates based on comments by @bdero.

### CODE REVIEW

File: bin/xbundle_convert [hunk 1]
```diff
@@ -1,69 +1,72 @@
...
         options['force_studio_format'] = True
 
-    cmd = sys.argv[argc]
-
-    if cmd == 'test':
+    if args['test']:
         run_tests()
+        return
+    if args['--version']:
+        print "xbundle {0}".format(get_distribution("xbundle").version)
```
Comment (bdero) [inline bin/xbundle_convert]: Thanks for finding out how to get the version!

docopt can actually handle the version flag, so it isn't actually necessary to check the flag/print the version and exit:
```python
args = docopt(
    __doc__,
    version="xbundle {}".format(get_distribution("xbundle").version)
)
```

File: bin/xbundle_convert [hunk 2]
```diff
@@ -1,69 +1,72 @@
...
+        print "done"
+    elif output_path.endswith('.xml'):
+        print "Converting edX directory '{0}' to xbundle '{1}'".format(
+            input_path, output_path)
+        bundle.import_from_directory(input_path)
+        bundle.save(output_path)
+        print "done"
     else:
-        usage()
+        print "Invalid input; run with --help for more info."
```
Comment (bdero) [inline bin/xbundle_convert]: Perhaps this should also cause an exit with a status code of `1` here after printing to indicate failure. E.g. `exit(1)`.

This is out of the scope of the PR, but if there are multiple points of failure deeper in the conversion process, we might want to think about having different status codes for different kinds of failures down the line if we'll be programmatically integrating this with other tools.

**Comment by ShawnMilo** [comment]
Updated version to use docopt's kwarg, added `sys.exit(1)` if arguments are invalid.

**Comment by bdero** [comment]
:electric_plug:

**Comment by pdpinch** [comment]
This will go in the first release unless you tell me otherwise. 

----------------
I am mobile.

**Pull request merged**